tutorials/ml_inference/Inference-in-SmartSim.ipynb

```diff
@@ -811,20 +824,21 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "id": "5daa7402-f62b-4710-a269-e078b2ce08ac",
    "metadata": {},
    "source": [
-    "# Co-Located Deployment\n",
+    "# CoLocated Deployment\n",
```

Comment:
Should we also make this "Colocated" (lower "l") just to be consistent with the rest of the titles/capitalizations of colocated?